Source/Core/UICommon/CommandLineParse.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "UICommon/CommandLineParse.h"
+#include "Common/Common.h"
+
+namespace CommandLineParse
+{
+std::unique_ptr<optparse::OptionParser> CreateParser(bool gui)
```

Comment:
`bool gui` should probably be an `enum class` or something to make parser creation more indicative at the callsite.

e.g.

```cpp
CreateParser(ParserOptions::IncludeGUIOptions);
```

is more informative than:

```cpp
CreateParser(true);
```